recipes/pastedeploy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "PasteDeploy" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d5858f89a255e6294e63ed46b73613c56e3b9a2d82a42f1df4d06c8421a9e3cb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - paste
+    - paste.deploy
+  requires:
```

Comment:
You can remove this `requires` section: we are not running the tests so it is not really needed.

Reply:
Removed